packages/components/models.json

```diff
@@ -673,6 +673,50 @@
                     "input_cost": 2.5e-7,
                     "output_cost": 1.25e-6
                 }
+            ],
+            "regions": [
```

Comment:
can we refractor the region into an object that can be re-used?

Reply:
Yes, no problem.  Do you have any preferences as to where in the code base?

Reply:
hmm i think we are good for now, because otherwise we can't make it into one single file